### ISSUE TITLE
Drupal composer ignores dependencies

### ISSUE THREAD

**Issue by miromichalicka** [acquia/commerce-manager#10]
**Description:**
```
⇒  composer show -i drupal/acquia_commercemanager
You are using the deprecated option "installed". Only installed packages are shown by default now. The --all option can be used to show all packages.
name     : drupal/acquia_commercemanager
descrip. : 
keywords : 
versions : * 1.1.0
type     : drupal-module
license  : GNU General Public License v2.0 or later (GPL-2.0-or-later) (OSI approved) https://spdx.org/licenses/GPL-2.0-or-later.html#licenseText
source   : [git] https://git.drupal.org/project/acquia_commercemanager 8.x-1.01
dist     : [zip] https://ftp.drupal.org/files/projects/acquia_commercemanager-8.x-1.01.zip 8.x-1.01
names    : drupal/acquia_commercemanager

support
source : http://cgit.drupalcode.org/acquia_commercemanager

requires
drupal/core ~8.0

requires (dev)
drupal/acm-acm *
drupal/acm-acm_checkout *
drupal/acm-acm_sku *
drupal/address *
drupal/field_group *
drupal/inline_entity_form *
drupal/key_value_field *
drupal/pcb *
drupal/simple_oauth *
drupal/simple_oauth_extras *
```

After adding this repository as vcs to composer.json, everything behaves as should.

```⇒  composer show --all drupal/acquia_commercemanager
name     : drupal/acquia_commercemanager                       
descrip. : This project provides the ability to use the Acquia Commerce Connector.
keywords : 
versions : dev-master, * 1.1.0, 1.0.0
type     : drupal-module
license  : GNU General Public License v2.0 or later (GPL-2.0+) (OSI approved) https://spdx.org/licenses/GPL-2.0+.html#licenseText
source   : [git] https://github.com/acquia/commerce-manager.git 2df7dad01d83233cef226609fc9116a4bde2d276
dist     : [zip] https://api.github.com/repos/acquia/commerce-manager/zipball/2df7dad01d83233cef226609fc9116a4bde2d276 2df7dad01d83233cef226609fc9116a4bde2d276
names    : drupal/acquia_commercemanager

support
source : https://github.com/acquia/commerce-manager/tree/1.1.0
issues : https://github.com/acquia/commerce-manager/issues

requires
commerceguys/intl ~0.7
drupal/address ~1.0
drupal/field_group ^1.0@RC
drupal/key_value_field ~1.0
acquia/http-hmac-php ~3.2
drupal/pcb ~1.0
drupal/simple_oauth 3.x-dev
```

**Comment by miromichalicka** [comment]
Issue opened on drupal.org https://www.drupal.org/project/project_composer/issues/2965650

**Comment by miromichalicka** [comment]
Attempt to fix in #11 .
